### ISSUE TITLE
Slimming down production app images

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/front-end-monorepo#1206]
Just as an experiment, I tried doing a production build of the project app and then trashing non-essential things. It turns out that this is all you need...

```sh
~/Code/zooniverse/front-end-monorepo/packages/app-project master*
$ ls -la
total 4
drwxr-xr-x  5 rogerhutchings staff  160 Oct  3 09:49 ./
drwxr-xr-x 14 rogerhutchings staff  448 Aug 16 18:41 ../
drwxr-xr-x  9 rogerhutchings staff  288 Oct  3 09:40 .next/
drwxr-xr-x 36 rogerhutchings staff 1152 Oct  1 10:13 node_modules/
-rw-r--r--  1 rogerhutchings staff 2528 Oct  1 17:31 package.json
```

cc'ing @eatyourgreens as it may be of interest

**Comment by eatyourgreens** [comment]
That's good to know. I found that if I restricted the file copying here to just `/packages/app-project` then `@zooniverse/classifier` and all the other linked libraries couldn't be resolved.
https://github.com/zooniverse/front-end-monorepo/blob/591aa59b19ca3fd35ad2f1e6f999b798a423b073/Dockerfile#L20-L24

Another problem I ran into while building production is that `git rev-parse` here will die if it isn't run inside a valid git repo.
https://github.com/zooniverse/front-end-monorepo/blob/bde032b4f76f1d94b28e0cb4bd88ea8c1fef9d7d/packages/app-project/next.config.js#L21-L26
